compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1784,8 +1783,7 @@ TR::Register *OMR::X86::TreeEvaluator::csubEvaluator(TR::Node *node, TR::CodeGen
       }
 
    TR::Register * testRegister = secondChild->getRegister();
-   if (secondChild->getOpCodeValue() == TR::cconst &&
```

Comment:
This whole if statement can be removed.  The condition is always `false` as the `TR::cconst` no longer exists

Reply:
I think this may need to be changed to sconst - I don't think it can be completely removed.

Reply:
Fixed in c215804